src/dal/Jalasoft.TeamUp.Projects.DAL/ProjectsRepository.cs

```diff
@@ -58,12 +58,20 @@ public class ProjectsRepository : IProjectsRepository
                 }
             };
 
+        private static IEnumerable<Project> projects = new List<Project>();
```

Comment:
please remove this and use the line 11 field, make the necessary changes to it ;)

Reply:
done